### ISSUE TITLE
Readme encoder output_size not fully understood 

### ISSUE THREAD

**Issue by insigh** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#3]
In the code the encoder output size should be :
           (batch_size, encoded_image_size, encoded_image_size, 2048)=(batch_size, 14, 14, 2048 )
in the Readme description image : 
                   it is (batch_size, 14, 14, 4096 ) !=(batch_size, 14, 14, 2048 )
 is there something i miss or not understanding? Thanks for your valuable work and i am looking forward to your applying !

**Comment by insigh** [comment]
Is the 4096 from VGGnet ?

**Comment by insigh** [comment]
I think the resnet output dim size should be 2048, so , is the 4096 from VGGNet ?

**Comment by sgrvinod** [comment]
Thanks for noticing! I wrote the tutorial some days after I wrote the code, and I must have had 4096 in my mind for some reason.

You are right. The number in the code is correct. It is indeed 2048 channels at the end of the ResNet-101 (likewise for the ResNet-50 and ResNet-152).

I'll correct the numbers in the Readme tomorrow. Sorry about the confusion! 

(If I remember right, the VGGnet has 512, similar to the smaller ResNets.)
